### PR TITLE
CORDA-1844: Support for high throughput Observables shipped via RPC

### PR DESCRIPTION
**Pull request by vkolomeyko**
Not a proper fix for production code yet, but a testcase exposing the problem.
Also minor extension to RPCServer which will allow to shut it down quicker.

### CODE REVIEW

File: testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/RPCDriver.kt [hunk 1]
```diff
@@ -485,7 +486,7 @@ data class RPCDriverDSL(
                 configuration
         )
         driverDSL.shutdownManager.registerShutdown {
-            rpcServer.close()
+            rpcServer.close(1.millis)
```
Comment (mnesbit) [inline testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/RPCDriver.kt]: This does look very fragile

Comment (vkolomeyko) [reply testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/RPCDriver.kt]: I can add an optional parameter to `RPCDriver`, but if we are shutting down the whole driver we might just shut down RPCServer as quickly as possible. That was my reason for 1ms.

File: node/src/main/kotlin/net/corda/node/services/messaging/RPCServer.kt [hunk 2]
```diff
@@ -247,9 +243,10 @@ class RPCServer(
         }
     }
 
-    fun close() {
+    fun close(queueDrainTimeout: Duration = 1.hours) {
```
Comment (mnesbit) [inline node/src/main/kotlin/net/corda/node/services/messaging/RPCServer.kt]: 5 seconds?

File: testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/RPCDriver.kt (hunk 1 above)
Comment (mnesbit) [reply testing/node-driver/src/main/kotlin/net/corda/testing/node/internal/RPCDriver.kt]: OK make it configurable

**Pull request merged**